### PR TITLE
update readme now that the extension is on packagist

### PR DESCRIPTION
**Pull request by agallou**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,25 +6,8 @@
...
-Then add the ruler-extension to your dev requirements :
-
-```json
-{
-	"require-dev": {
-		"atoum/ruler-extension": "dev-master"
-	},
-}
-
+composer require atoum/ruler-extension:dev-master
```
Comment (jubianchi) [inline README.md]: `require --dev` ?

**Comment by agallou** [comment]
i've added the `--dev` option. When we'll create the first tag i'll remove the `:dev-master` version.

**Comment by Hywan** [comment]
Could be great to create a `0.1.0` version to get rid off the `--dev` (Composer does not like it :-(…).

**Comment by agallou** [comment]
@Hywan why get rid off the `--dev` option ? The library is for testing, so it will mostly be on the require_dev section of the composer.json. (but when will create a tag, we could remove the `:dev-master`).

**Comment by Hywan** [comment]
@agallou Yup ;-).

**Comment by jubianchi** [comment]
:+1: for me now.

One the first tag is pushed we'll only have to remove the `:dev-master` as @agallou said. I think we should merge this one as is and fix it later (when the tag is out)

**Comment by Hywan** [comment]
@jubianchi Exact.

**Pull request merged**